gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/openapi-apis.yaml

```diff
@@ -2919,6 +2968,45 @@ components:
             allOf:
                 - $ref: "#/components/schemas/UpdateGenericApi"
 
+        CategoryApi:
+          type: object
+          description: API in a category
+          required:
+            - id
+            - name
+            - version
```

Comment:
what is version here 🤔 ? apiVersion instead no ?

Reply:
good catch